doc/src/Intro_citing.rst

```diff
@@ -4,28 +4,40 @@ Citing LAMMPS
 Core Algorithms
 ^^^^^^^^^^^^^^^
 
-Since LAMMPS is a community project, there is not a single one
-publication or reference that describes **all** of LAMMPS.
-The canonical publication that describes the foundation, that is
-the basic spatial decomposition approach, the neighbor finding,
-and basic communications algorithms used in LAMMPS is:
+Since LAMMPS is a community project there is not one single publication
+or reference that describes **all** of LAMMPS.  There are publications
+describing the core of LAMMPS, but then also publications describing the
+algorithms and methods it implements or the software that it has
+interfaces to.  Please see below for how to cite contributions to
+LAMMPS.
 
- `S. Plimpton, Fast Parallel Algorithms for Short-Range Molecular Dynamics, J Comp Phys, 117, 1-19 (1995). <http://www.sandia.gov/~sjplimp/papers/jcompphys95.pdf>`_
+.. _lammps_paper:
 
-So any project using LAMMPS (or a derivative application using LAMMPS as
-a simulation engine) should cite this paper. A new publication
-describing the developments and improvements of LAMMPS in the 25 years
-since then is currently in preparation.
+The canonical publication that describes the basic features, the source
+code design, the program structure, the spatial decomposition approach,
+the neighbor finding, basic communications algorithms, and how users and
+developers have contributed to LAMMPS is:
+
+  `LAMMPS - A flexible simulation tool for particle-based materials modeling at the atomic, meso, and continuum scales, Comp. Phys. Comm. (accepted 09/2021), DOI:10.1016/j.cpc.2021.108171 <https://doi.org/10.1016/j.cpc.2021.108171>`_
+
+So any project using LAMMPS or a derivative application that uses LAMMPS
+as a simulation engine should cite this paper.  The paper is expected to
+be published in its final form under the same DOI in the first half of 2022.
+
+The original publication describing LAMMPS in its early implementations is:
```

Comment:
The original pub describing the parallel algorithms used in the initial versions of LAMMPS is: